Prolific/Prolific/Utils/Persistence/DAO.h

```diff
@@ -78,6 +72,11 @@ NS_ASSUME_NONNULL_BEGIN
 
 #pragma mark - Project
 
+/** Updates a project with a new follower or unfollower. Will return error message if update unsuccessful. */
+- (void)updateFollowersforProject:(Project *)project
```

Comment:
does this api already know the current follow state somehow?

Reply:
I see further down it looks at this internally. I'd say you probbaly want to have to methods `addFollowerForProject` and `removeFollowerForProject` 

Think of your API as an ad for what your class does. With this the caller doesn't know explicitly what is going to happen